### ISSUE TITLE
custom fonts don't work properly

### ISSUE THREAD

**Issue by choojang** [ZhenjianYang/SoraVoice#7]
I'm playing trails of sky fc(eng steam) with my own fonts modified chinese characters.
and after jan patch, I updated dll files but fonts seem to been loaded but my modified characters doesn't printed properly in game and they look like crashed.
![crashed](https://user-images.githubusercontent.com/101637017/158366723-daae21cd-9905-46b8-b510-8357e40dbc7f.png)
.

**Comment by ZhenjianYang** [comment]
Hi,
How do you modify the fonts?
Can you share more information?

**Comment by choojang** [comment]
I modified some chinese characters of original font by hex editor and when I play with 2021-7-12 steam beta version they printed correctly now. but with last version, even though I put original fonts not modified in voice/fonts folder, I got the same result characters crashed.

**Comment by ZhenjianYang** [comment]
Sounds not like an issue of the voice patch.
Do your fonts work well without the voice patch?

**Comment by choojang** [comment]
Yes, my fonts worked well in ed6_dt00.dat, but when I tried to load them in voice/fonts with latest soravoice, they didn't.

**Comment by ZhenjianYang** [comment]
Hi,
Loading fonts from voice/fonts has not been fully supported in this Lite version.
I'm afraid you have to modify ed6_dt00.dat directly.
